<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>clientWidget</class>
 <widget class="QWidget" name="clientWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>790</width>
    <height>690</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>clientWidget</string>
  </property>
  <widget class="QStackedWidget" name="stackedWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>790</width>
     <height>690</height>
    </rect>
   </property>
   <widget class="QWidget" name="index">
    <widget class="QPushButton" name="five">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>90</y>
       <width>141</width>
       <height>101</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>30</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Five</string>
     </property>
    </widget>
    <widget class="QPushButton" name="go">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>90</y>
       <width>141</width>
       <height>101</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>30</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Go</string>
     </property>
    </widget>
    <widget class="QPushButton" name="exit">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>260</y>
       <width>75</width>
       <height>35</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>20</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Exit</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="fiveWidget">
    <widget class="QPushButton" name="five_exit">
     <property name="geometry">
      <rect>
       <x>690</x>
       <y>70</y>
       <width>75</width>
       <height>75</height>
      </rect>
     </property>
     <property name="text">
      <string>Exit</string>
     </property>
    </widget>
    <widget class="ChessBoard" name="board_five">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>20</y>
       <width>571</width>
       <height>441</height>
      </rect>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
    <widget class="QLabel" name="five_label">
     <property name="geometry">
      <rect>
       <x>710</x>
       <y>160</y>
       <width>101</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>12</pointsize>
       <weight>75</weight>
       <bold>true</bold>
       <kerning>true</kerning>
      </font>
     </property>
     <property name="text">
      <string>It's your turn</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="goWidget">
    <widget class="QPushButton" name="go_exit">
     <property name="geometry">
      <rect>
       <x>700</x>
       <y>80</y>
       <width>75</width>
       <height>75</height>
      </rect>
     </property>
     <property name="text">
      <string>Exit</string>
     </property>
    </widget>
    <widget class="ChessBoard" name="board_go">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>30</y>
       <width>571</width>
       <height>441</height>
      </rect>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
    <widget class="QLabel" name="go_label">
     <property name="geometry">
      <rect>
       <x>730</x>
       <y>210</y>
       <width>101</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>12</pointsize>
       <weight>75</weight>
       <bold>true</bold>
       <kerning>true</kerning>
      </font>
     </property>
     <property name="text">
      <string>It's your turn</string>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>ChessBoard</class>
   <extends>QLabel</extends>
   <header location="global">chessboard.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
